src/parsing.jl

```diff
@@ -319,7 +321,8 @@ function _interval_regex()
     for long_type in (Hour, Minute)
         print(io, _field_match(long_type))
     end
-    print(io,
+    print(
+        io,
```

Comment:
:|

Reply:
yeah. that's just what BlueStyle suggests. 🤷 but i get the :| the sentament

Reply:
I don't get why this is forced but [here](https://github.com/invenia/LibPQ.jl/pull/197#discussion_r492824309) the trailing argument is allowed on the same line as the closing parenthesis.